src/coreclr/src/tools/crossgen2/crossgen2/Program.cs

```diff
@@ -258,17 +237,51 @@ private int Run()
             optimisticInstructionSet.Remove(unsupportedInstructionSet);
             optimisticInstructionSet.Add(supportedInstructionSet);
 
-            var instructionSetSupport = new InstructionSetSupport(supportedInstructionSet,
+            return new InstructionSetSupport(supportedInstructionSet,
                                                                   unsupportedInstructionSet,
                                                                   optimisticInstructionSet,
                                                                   InstructionSetSupportBuilder.GetNonSpecifiableInstructionSetsForArch(_targetArchitecture),
                                                                   _targetArchitecture);
+        }
 
+        private void CheckCustomPESectionAlignment()
+        {
+            if (_commandLineOptions.CustomPESectionAlignment != null)
+            {
+                int alignment = _commandLineOptions.CustomPESectionAlignment.Value;
+                bool invalidArgument = false;
+                if (alignment <= 4096)
```

Comment:
Why is 4096 considered invalid? I was under the impression that is the default for Windows PE files. Should we be using a "less then" comparison sign (<code>&lt;</code> instead of <code>&lt;=</code>)?

Reply:
Yeah that looks fishy. @davidwrighton this should be `if (alignment < 4096)` right?

Reply:
This command line switch should only be used for out of standard alignment PE files. Arguably it should be something like customHugePESectionAlignmentSize to make it clear it isn't associated with normal PE alignment.